### PR TITLE
Refactor

### LINKED ISSUE #68: Refactor code to Contact
There is a need to change all instances of "Person" to "Contact" to align with the project user guide.

### PR DESCRIPTION
**Pull request by pcgiang**
Fix #68 
- [ ] rename files and packages
- [ ] add new packages for commands

### CODE REVIEW

**Review by chunweii — CHANGES_REQUESTED** [review]
Only change Java files! Please roll back the changes and refactor again.

File: src/test/java/seedu/address/logic/commands/general/CommandTestUtil.java [hunk 1]
```diff
@@ -75,7 +78,7 @@
      * - the {@code actualModel} matches {@code expectedModel}
      */
     public static void assertCommandSuccess(Command command, Model actualModel, CommandResult expectedCommandResult,
-            Model expectedModel) {
+                                            Model expectedModel) {
```
Comment (xiangjunn) [inline src/test/java/seedu/address/logic/commands/general/CommandTestUtil.java:81]: extra indentation. see `line 96`

File: src/test/java/seedu/address/logic/commands/contact/CAddCommandIntegrationTest.java [hunk 2]
```diff
@@ -10,13 +10,13 @@
 import seedu.address.model.Model;
 import seedu.address.model.ModelManager;
 import seedu.address.model.UserPrefs;
-import seedu.address.model.person.Person;
+import seedu.address.model.contact.Contact;
 import seedu.address.testutil.PersonBuilder;
 
 /**
  * Contains integration tests (interaction with the Model) for {@code AddCommand}.
```
Comment (xiangjunn) [inline src/test/java/seedu/address/logic/commands/contact/CAddCommandIntegrationTest.java:17]:
```suggestion
 * Contains integration tests (interaction with the Model) for {@code CAddCommand}.
```

File: src/main/java/seedu/address/storage/JsonAdaptedContact.java [hunk 3]
```diff
@@ -31,12 +31,12 @@
...
-     * Constructs a {@code JsonAdaptedPerson} with the given person details.
+     * Constructs a {@code JsonAdaptedContact} with the given contact details.
      */
     @JsonCreator
-    public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
-            @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+    public JsonAdaptedContact(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
+                              @JsonProperty("email") String email, @JsonProperty("address") String address,
+                              @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
```
Comment (xiangjunn) [inline src/main/java/seedu/address/storage/JsonAdaptedContact.java:39]: should indentation for wrapped lines be 8 spaces instead?

File: src/main/java/seedu/address/logic/parser/contact/CEditCommandParser.java [hunk 4]
```diff
@@ -14,22 +14,25 @@
...
+import seedu.address.logic.commands.contact.CEditCommand;
+import seedu.address.logic.parser.ArgumentMultimap;
+import seedu.address.logic.parser.ArgumentTokenizer;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.ParserUtil;
 import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.tag.Tag;
 
 /**
  * Parses input arguments and creates a new EditCommand object
```
Comment (xiangjunn) [inline src/main/java/seedu/address/logic/parser/contact/CEditCommandParser.java:26]:
```suggestion
 * Parses input arguments and creates a new CEditCommand object
```

**Review by xiangjunn — CHANGES_REQUESTED** [review]
Some small changes to be made. Other than that, LGTM!

File: src/main/java/seedu/address/storage/JsonAdaptedContact.java (hunk 3 above)
Comment (chunweii) [reply src/main/java/seedu/address/storage/JsonAdaptedContact.java:39]: This is the default style that we already have since the start

**Pull request merged**